app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(players: list) -> int:
+    return sum([player.get_rating() for player in players])
+
+
+def elves_concert(elves: list) -> None:
```

Comment:
It’s better to write exactly what you expect in the `list` . Please specify the correct typing